src/main/java/entities/Game.java

```diff
@@ -56,6 +57,15 @@ public Card getCurrentCard() {
         return this.currentCard;
     }
 
+    /**
+     * Get the Card that was drawn last.
+     * @return A Card object which the current Player has drawn from the deck.
+     */
+    @Override
+    public Card getDrawnCard() {
```

Comment:
We should an issue to have a test case for this method.

Reply:
agreed